samples/ScreenTFT_S1D13781/app/app.cpp

```diff
@@ -0,0 +1,287 @@
+#include <SmingCore.h>
```

Comment:
- Please rename this file to app/application.cpp to be consistent with the other samples that we have.
- Remove the anonymous namespace. Adding an anonymous namespace to all samples should happen in a later release.

Reply:
OK, I've moved the sample into the library and stripped out all the RAM stuff so it's much simpler.